apollo/graph.py

```diff
@@ -279,7 +279,8 @@ def __call__(self, graph):
             kwargs = {"weights": graph.edge_weights}
         if self.algorithm == "edge_betweenness":
             kwargs["directed"] = False
-        result = action(**kwargs, **self.config)
+        kwargs.update(self.config)
+        result = action(**kwargs)
```

Comment:
Please put TODO here which explains that Python 3.4 sucks. Once we move to Ubuntu 18.04, we stop supporting 14.04 and thus 3.4 and remove this hack.